package.json

```diff
@@ -627,6 +627,7 @@
   "dependencies": {
     "@kubernetes/client-node": "^0.8.1",
     "byline": "^5.0.0",
+    "copy-paste": "^1.3.0",
```

Comment:
I think we can use a better npm module than this which is maintained or has more downloads atleast.